openrouteservice-api-tests/src/test/java/org/heigit/ors/services/mapmatching/ResultsValidationTest.java

```diff
@@ -20,16 +20,8 @@
  */
 package org.heigit.ors.services.mapmatching;
 
-import static io.restassured.RestAssured.*;
-
-import org.junit.Test;
-import org.json.JSONArray;
-import org.json.JSONObject;
-
```

Comment:
were these imports not used anyway?
I can't check right now, but if you want this merged before monday in 2 weeks, maybe let someone else look :)
Otherwise i will do on the 19th